### PR TITLE
fix: error when create funnel type chart

### PR DESCRIPTION
**Pull request by llmin**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

fix a bug when  create funnel type chart

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [x] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/control-plane/backend/lambda/api/test/api/reporting.test.ts [hunk 1]
```diff
@@ -389,6 +389,101 @@ describe('reporting test', () => {
...
+            dataApiRole: 'arn:aws:iam::11111111:role/test_api_role',
+            newServerless: {
+              workgroupName: 'clickstream-project01-wvzh',
+            },
+          },
+        },
+      });
+
+    expect(res.headers['content-type']).toEqual('application/json; charset=utf-8');
+    expect(res.statusCode).toBe(500);
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/test/api/reporting.test.ts:484]: We should catch this exception then response 400 bad request for invalid chart type.

**Review by zxkane — CHANGES_REQUESTED** [review]

File: src/control-plane/backend/lambda/api/test/api/reporting.test.ts [hunk 2]
```diff
@@ -389,6 +389,101 @@ describe('reporting test', () => {
     expect(res.body.data.visualIds.length).toEqual(2);
   });
 
+  it('funnel visual - error chart type', async () => {
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/test/api/reporting.test.ts:392]: -> unsupported / invalid chart type

**Review by zxkane — CHANGES_REQUESTED** [review]

File: src/control-plane/backend/lambda/api/test/api/reporting.test.ts (hunk 2 above)
Comment (llmin) [reply src/control-plane/backend/lambda/api/test/api/reporting.test.ts:392]: fixed

File: src/control-plane/backend/lambda/api/test/api/reporting.test.ts (hunk 1 above)
Comment (llmin) [reply src/control-plane/backend/lambda/api/test/api/reporting.test.ts:484]: will add a task to  check all parameters in backend and commit in another PR.

**Review by zxkane — APPROVED** [review]

**Pull request merged**